backoff.py

```diff
@@ -218,10 +218,22 @@ def constant(interval=1):
         yield interval
 
 
+def random_jitter(value):
+    return value+random.random()
```

Comment:
This line looks like it would fail pep8 check. You probably just need one last `make check` on your code.